tests/jax/integration/data_server_test.py

```diff
@@ -12,91 +12,134 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+"""Tests for data server for Jax-based Mava systems"""
 
-"""Tests for config class for Jax-based Mava systems"""
-from typing import Any, Dict
-
+import jax.numpy as jnp
 import pytest
-from reverb import item_selectors, pybind, reverb_types
+import reverb
+from tensorflow.python.data.ops import dataset_ops
+
+from mava.systems.jax import System
+from tests.jax.integration.mock_systems import mock_system_single_process
+
+
+@pytest.fixture
+def test_system() -> System:
+    """A built IPPO system"""
+    return mock_system_single_process()
```

Comment:
Thanks, @OmaymaMahjoub :slightly_smiling_face:  Looks good. Are we only testing multi_process for the integration tests?

Reply:
Thanks @DriesSmit  for reviewing! Unfortunately since data server a Reverb node it was tough to find out how to test it in the multi process so at least we tested it in the single process case

Reply:
That makes sense.